website/website/add_youtuber.py

```diff
@@ -0,0 +1,77 @@
+from googleapiclient.discovery import build
+from urllib.parse import urlparse, unquote
+import os
+API_KEY = os.environ.get('API_KEY')
```

Comment:
Let's avoid this constant on the module level, if it is used only on a single class level.

Besides, you already declare it on the class constructor level.